cmake/readerTpl.h.in

```diff
@@ -5,7 +5,7 @@
 #ifndef F3D@F3D_READER_NAME@_h
```

Comment:
Tpl stand for template ? why not using the term boiler plate ? and shy shortening it ?

Reply:
I prefer the term `Template` but if you prefer `BoilerPlate` I'm fine. I will restore it.

Reply:
template is very C++ charged imo. as you prefer, no reason to shorten it though.